### ISSUE TITLE
Docs generation is failing

### ISSUE THREAD

**Issue by andreaTP** [andreaTP/app-services-backstage#3]
The generation of docs for imported projects fails with:
```
Failed to build the docs page: Failed to generate docs from /tmp/backstage-lQYFEa into /tmp/techdocs-tmp-c59zot; caused by Error: This operation requires Docker. Docker does not appear to be available. Docker.ping() failed with; caused by Error: connect ENOENT /var/run/docker.sock {"timestamp":"2022-08-22T17:06:10.590Z"}
```

**Comment by andreaTP** [comment]
Now missing `mkdocs` package:
```
error: Failed to build the docs page: Failed to generate docs from /tmp/backstage-OkJu5X into /tmp/techdocs-tmp-6JNgR8; caused by Error: spawn mkdocs ENOENT {"timestamp":"2022-08-22T17:44:21.671Z"}
```

**Comment by wtrocki** [comment]
More context

**Comment by wtrocki** [comment]
We do not have any documentation in tech doc format so I would not see that being useful..

Most important parts are api/components and generators.

**Comment by andreaTP** [comment]
It should be fixed in #5

**Comment by andreaTP** [comment]
Fixed: https://backstage-apicurio-apicurio-registry.apps.smaug.na.operate-first.cloud/catalog/default/component/backstage/docs

Thanks @jsenko for rising the issue
